android/build.gradle

```diff
@@ -2,14 +2,14 @@ group 'com.amplitude.amplitude_flutter'
 version '1.0-SNAPSHOT'
 
 buildscript {
-    ext.kotlin_version = '1.7.10'
+    ext.kotlin_version = '1.5.20'
```

Comment:
Are we going to an earlier version of kotlin?

Reply:
I think I lowered it for some compatibility issues. I don't see any harms to customers to lower the version. Do you any concern? I'm open to discussion and can try to see if I can revert it if it's necessary.